files/src/config/module-map.ts

```diff
@@ -1,6 +0,0 @@
-/**
```

Comment:
This is somewhat moot now, I brought this back as a *.d.ts only. This prevents editors from warning about this module missing, but also makes it clear that its only a definition file...